client/request/consts.go

```diff
@@ -21,6 +21,8 @@ const (
 
 	Cid         = "cid"
 	Input       = "input"
+	CreateInput = "create"
+	UpdateInput = "update"
```

Comment:
question: shouldn't "create" and "update" replace "input"?

Reply:
I think keeping the `input` label for create and update mutations is fine.

Changing it to `create_User(create: {name: "Bob"})` doesn't read as nice imo.